pages/posts/2.tsx

```diff
@@ -0,0 +1,179 @@
+import Head from "next/head"
+import Navbar from "../components/Navbar/Navbar"
+import Footer from "../components/Footer/Footer"
+import styles from "../../styles/Home.module.scss"
+import content from "./content.json"
+import Prism from "prismjs"
+import React from "react"
+
+export default function Post() {
+  let posts
+  posts = content.items[1]
+  console.log(posts)
+
+  React.useEffect(() => {
+    const highlight = async () => {
+      await Prism.highlightAll() // <--- prepare Prism
+    }
+    highlight() // <--- call the async function
+  }, [])
+
+  return (
+    <>
+      <Head>
+        <title>Wesh Network</title>
+        <meta name="description" content="Wesh Network" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <main className={styles.Body}>
+        <Navbar />
+
+        <section className={styles.SectionBlog}>
+          <div className={styles.SectionBlogContainer}>
+            <div className={styles.SectionBlogCard}>
+              <div className={styles.SectionBlogText}>
+                <h2>{posts.title}</h2>
+                <p>
+                  Now that you have been introduced to Wesh in the
+                  <a href="https://wesh.network/posts/1"> previous blog post</a>, it’s time to explore the code. We will write a “Hello World” app, similar to the
+                  <a href="https://go.dev/doc/tutorial/getting-started"> Go tutorial </a>
+                  which you can read to set up Go on your computer and for other details. (This has been tested with Go 1.19 on macOS and Ubuntu.) This example app will connect to
+                  the Wesh service and call a function. When finished, you will be confident that you can use Wesh on your platform.
+                </p>
+                <p>Similar to the Go tutorial, in a terminal enter:</p>
+                <pre>
+                  <code className="language-js">
+                    {`
+$ cd mkdir hello
```

Comment:
This should be on two lines:

    cd
    mkdir hello
    
(The first cd goes to the user's home directory.)